pkg/compare/httpfs.go

```diff
@@ -16,7 +16,7 @@ const defaultHttpGetAttempts = 5
 
 // isURL checks if the given path is a URL by verifying if it starts with "http://" or "https://".
 func isURL(path string) bool {
-	return strings.Index(path, "http://") == 0 || strings.Index(path, "https://") == 0
+	return strings.HasPrefix(path, "http://") || strings.HasPrefix(path, "https://")
```

Comment:
@nocturnalastro i copied it from here https://github.com/kubernetes/cli-runtime/blob/master/pkg/resource/builder.go#L262

so if you want you can change it there too :laughing: 

I think the new option is better :)